files/system/usr/share/ublue-os/just/70-secureblue.just

```diff
@@ -634,35 +634,51 @@ audit-secureblue:
 
             if hasPermission "$permissions" "shared" "network"; then
                 [[ "$status" != "$STATUS_FAILURE" ]] && status="$STATUS_WARNING"
-                warnings+=("> $f has network access!")
+                warnings+=("> $f has network access!
+>To remove it use Flatseal or run:
+> 'flatpak override -u --unshare=network $f'")
             fi
             if hasPermission "$permissions" "sockets" "x11" && ! hasPermission "$permissions" "sockets" "fallback-x11"; then
                 status="$STATUS_FAILURE"
-                warnings+=("> $f has x11 access!")
+                warnings+=("> $f has x11 access!
+> To remove it use Flatseal or run:
+> 'flatpak override -u --nosocket=x11 $f'")
             fi
             if hasPermission "$permissions" "sockets" "session-bus"; then
                 [[ "$status" != "$STATUS_FAILURE" ]] && status="$STATUS_WARNING"
-                warnings+=("> $f has access to the D-Bus session bus!")
+                warnings+=("> $f has access to the D-Bus session bus!
+> To remove it use Flatseal or run:
+> 'flatpak override -u --nosocket=session-bus $f'")
             fi
             if hasPermission "$permissions" "sockets" "system-bus"; then
                 [[ "$status" != "$STATUS_FAILURE" ]] && status="$STATUS_WARNING"
-                warnings+=("> $f has access to the D-Bus system bus!")
+                warnings+=("> $f has access to the D-Bus system bus!
+> To remove it use Flatseal or run:
+> 'flatpak override -u --nosocket=system-bus $f'")
             fi
             if ! hasPermission "$permissions" "LD_PRELOAD" .*"/libhardened_malloc.so"; then
                 status="$STATUS_FAILURE"
-                warnings+=("> $f is not using hardened_malloc!")
+                warnings+=("> $f is not using hardened_malloc!
+> To enable it run:
+> 'ujust harden-flatpak'")
```

Comment:
I just realized this actually won't work as a suggestion since it will just reapply the global override. if the flatpak in question has it removed, the global override wont reapply it

So this would need to be adjusted, and also it points to potentially needing another check (for the global override)

Reply:
Instead of suggesting either right here on the audit-secureblue script, there could be a separate ujust script that applies the hwcaps command to a specified flatpak, and such command could be suggested here instead.

Reply:
good point. #546 adds an optional parameter to harden-flatpak target a specific flatpak.